packages/components/addon/modifiers/hds-clipboard.js

```diff
@@ -97,8 +97,9 @@ export const writeTextToClipboard = async (textToCopy) => {
       return true;
     } catch (error) {
       // clipboard write failed
-      console.log(
-        '`hds-clipboard` modifier - an error occurred while writing to the clipboard - check your browser permissions',
+      // this probably never happens (see comment above) or happens only for very old browsers that don't for which `navigator.clipboard` is undefined
+      warn(
+        'copy action failed, please check your browser‘s permissions',
         textToCopy,
         error
       );
```

Comment:
[`Ember.warn`s API](https://api.emberjs.com/ember/release/functions/@ember%2Fdebug/warn) is different than `console.log`'s – it seems to be at least requiring an object with an identifier as a third argument.

Reply:
Ohhh, I see! Fixed, let's see if the tests are passing now...